op-node/metrics/metrics.go

```diff
@@ -491,8 +500,25 @@ func (m *Metrics) RecordGossipEvent(evType int32) {
 	m.GossipEventsTotal.WithLabelValues(pb.TraceEvent_Type_name[evType]).Inc()
 }
 
+func getLabelForScore(score float64) string {
+	if score <= -40 {
+		return "banned"
+	} else if score <= 0 {
+		return "low"
+	} else if score <= 20 {
+		return "medium"
+	} else {
+		return "high"
+	}
+}
+
 func (m *Metrics) RecordPeerScoring(peerID peer.ID, score float64) {
-	m.PeerScores.WithLabelValues(peerID.String()).Set(score)
+	prevScore, ok := m.PeerStore[peerID]
+	if ok {
+		m.PeerScores.WithLabelValues(getLabelForScore(prevScore)).Dec()
+	}
+	m.PeerScores.WithLabelValues(getLabelForScore(score)).Inc()
```

Comment:
Could be improved not to update if score stays the same. Otherwise there would be a decrease followed by an increase right after.